core/src/view/view.cpp

```diff
@@ -134,7 +135,12 @@ void View::setRoll(float _roll) {
 
 void View::setPitch(float _pitch) {
 
-    m_pitch = glm::clamp(_pitch, 0.f, (float)HALF_PI);
+    float max_pitch = HALF_PI;
+    if (m_type != CameraType::perspective) {
+        // Prevent projection plane from intersecting ground plane
+        max_pitch = atan2(m_pos.z, m_height * .5f);
```

Comment:
I think this might not be fully working.. Check the following image, I got to this by moving from landscape view to portrait view. Hence maybe ... this should be `max(m_height, m_width)`?

![screenshot_2015-10-21-15-41-20](https://cloud.githubusercontent.com/assets/360641/10648105/a18b39bc-780a-11e5-8ef5-be77ea936b6d.png)